### PR TITLE
Enable assay library import for small molecules

### PR DESCRIPTION
**Pull request by bspratt**

### CODE REVIEW

File: pwiz_tools/Skyline/Model/Import.cs [hunk 1]
```diff
@@ -411,6 +419,8 @@ public static bool TryInitFormat(string inputLines, out IFormatProvider provider
             return MassListImporter.IsColumnar(inputLines, out provider, out sep, out var columnTypes);
         }
 
+        public int? LineCountLimit { get; set; } // Process no more than LineCountLimit lines when set
```
Comment (nickshulman) [inline pwiz_tools/Skyline/Model/Import.cs:422]: This never actually gets set anywhere. Is that intentional?

**Review by nickshulman — APPROVED** [review]
Looks good to me.

File: pwiz_tools/Skyline/Model/Import.cs (hunk 1 above)
Comment (bspratt) [reply pwiz_tools/Skyline/Model/Import.cs]: Nah, just cruft. Removed, thanks.

File: pwiz_tools/Skyline/FileUI/ChooseIrtStandardPeptidesDlg.cs [hunk 2]
```diff
@@ -204,11 +212,11 @@ private bool ProcessProtein()
...
 
             var protein = ((PeptideGroupItem)comboProteins.SelectedItem).PeptideGroup;
-            SetStandards(protein.Peptides.Select(pep => pep.ModifiedTarget));
-            if (Document.PeptideGroupCount > 0)
+            SetStandards(protein.Molecules.Select(pep => pep.ModifiedTarget));
+            if (Document.MoleculeGroupCount > 0)
             {
                 var pathFrom = Document.GetPathTo(Document.FindNodeIndex(protein.Id));
-                var pathTo = Document.GetPathTo(Document.FindNodeIndex(Document.PeptideGroups.First().Id));
+                var pathTo = Document.GetPathTo(Document.FindNodeIndex(Document.MoleculeGroups.First().Id));
```
[the remark this answers is not in the document]
Comment (bspratt) [reply pwiz_tools/Skyline/FileUI/ChooseIrtStandardPeptidesDlg.cs]: OK, changed as recommended.

**Pull request merged**